src/domain/app/i18n/en.json

```diff
@@ -8,6 +8,16 @@
   },
   "appDescription": "Linked Events is a collection of software components and API endpoints that enables event management and distribution for different event providers in Finland.",
   "appName": "Linked Events",
+  "attendanceListPage": {
+    "count": "{{count}} attendee",
+    "count_other": "{{count}} attendees",
```

Comment:
Nitpicking: following naming convention should this be `countOther` instead? 🤔

Reply:
`count_other` is actually used to show plural form of translation of key `count`. See https://www.i18next.com/translation-function/plurals#singular-plural